include/baresip.h

```diff
@@ -419,11 +419,13 @@ struct config_avt {
 	bool rtcp_mux;          /**< RTP/RTCP multiplexing          */
 	struct {
 		enum jbuf_type jbtype;  /**< Jitter buffer type     */
-		struct range jbuf_del;  /**< Delay, number of frames*/
+		struct range jbuf_del;  /**< Max./Min. Delay [ms]   */
+		uint32_t jbuf_sz;       /**< Max. buffer  [packets] */
 	} audio;
 	struct {
 		enum jbuf_type jbtype;  /**< Jitter buffer type     */
-		struct range jbuf_del;  /**< Delay, number of frames*/
+		struct range jbuf_del;  /**< Max./Min. Delay [ms]   */
+		uint32_t jbuf_sz;       /**< Max. buffer  [packets] */
 	} video;
```

Comment:
The audio and video structs look similar ...

Reply:
Yes, but audio and video need different parameters (audio/video latency or max packet settings). Or do you mean to create a struct for reuse?

Reply:
It is possible to write the code like this for reuse:

```
struct {
    int foo;
    int bar;
    /* etc.. */
} audio, video;
```